_integration/testsuite/README.md

```diff
@@ -0,0 +1,19 @@
+# Contour integration tests
+
+For now, run integration tests manually.
+
+The [make-kind-cluster.sh](./make-kind-cluster.sh) script will bring up
+a local kind cluster. It will build Contour from the working repository
+and install it, along with [cert-manager](https://cert-manager.io),
+which is needed for tests that use TLS.
+
+You will need to install the [Integration Tester for Kubernetes](https://github.com/projectcontour/integration-tester).
```

Comment:
I was going to test this, but there's nothing to install just yet? https://github.com/projectcontour/integration-tester

Reply:
I see there's a PR for that.

Reply:
Yup :)